ergo/platforms/predictit.py

```diff
@@ -0,0 +1,332 @@
+        """
+        Return the lastTradePrice
+        """
+        return self.lastTradePrice
+
+    def refresh(self):
+        """
+        Refetch the market data from PredictIt and reload the question.
+        """
+        self.market.refresh()
+        self._data = self.market.get_question_by_id(self.id)._data
+
+    def sample_community(self) -> bool:
+        """
+        Sample from the PredictIt community distribution (Bernoulli).
+        :return: true/false
+        """
+        return flip(self.get_community_prediction())
+
+
+class PredictItMarket:
+    """
+        A PredictIt market.
+
+        :param predictit: PredictIt API instance
+        :param data: Market JSON retrieved from PredictIt API
+
+        :ivar PredictIt predictit: predictit api instance
+        :ivar str api_url: url of the predictit api for the given question
+        :ivar int id: id of the market
+        :ivar str name: name of the market
+        :ivar str shortName: shortened name of the market
+        :ivar str image: url of the image resource of the market
+        :ivar str url: url of the market in PredictIt
+        :ivar str status: status of the market. Closed markets aren't included in the API, so always "Open"
+        :ivar datetime.datetime timestamp: last time the market was updated.
+            Api updates every minute, but timestamp can be earlier if it hasn't been traded in
+    """
+
+    def __init__(self, predictit: Any, data: Dict):
+        self.predictit = predictit
+        self._load_attr(data)
+
+    def __repr__(self):
+        if self._data:
+            return f'<PredictItMarket title="{self.name}">'
+        else:
+            return "<PredictItMarket>"
+
+    def __str__(self):
+        return repr(self)
+
+    def _load_attr(self, data):
+        """
+        Load attributes of question from data to instance.
+        :param data:
+        """
+        self._data = data
+        self.id = data['id']
+        self.name = data['name']
+        self.shortName = data['shortName']
+        self.image = data['image']
+        self.url = data['url']
+        self.status = data['status']
+        self.api_url = f"{self.predictit.api_url}/markets/{self.id}/"
+        if data['timeStamp'] == 'N/A':
+            self.timestamp = None
+        else:
+            self.timestamp = parse(data['timeStamp'])
+
+    @property
+    def questions(self) -> Iterator[PredictItQuestion]:
+        """
+        Generate all of the questions in the market.
+        :return: iterator of questions in market
+        """
+        for data in self._data['contracts']:
+            yield PredictItQuestion(self, data)
+
+    def refresh(self):
+        """
+        Refetch the market data from PredictIt,
+        used when the question data might have changed.
+        """
+        r = self.predictit._get(self.api_url)
+        self._load_attr(r.json())
+
+    def get_question_by_id(self, id: int) -> PredictItQuestion:
```

Comment:
If there's not other method for getting questions, can just call it `get_question`.